unblob/handlers/compression/lzh.py

```diff
@@ -14,25 +14,22 @@ class LZHHandler(StructHandler):
 
     NAME = "lzh"
 
-    YARA_RULE = r"""
-        strings:
-            $lzh_magic_lh0 = "-lh0-"
-            $lzh_magic_lzs = "-lzs-"
-            $lzh_magic_lz4 = "-lz4-"
-            $lzh_magic_lh1 = "-lh1-"
-            $lzh_magic_lh2 = "-lh2-"
-            $lzh_magic_lh3 = "-lh3-"
-            $lzh_magic_lh4 = "-lh4-"
-            $lzh_magic_lh5 = "-lh5-"
-            $lzh_magic_lh6 = "-lh6-"
-            $lzh_magic_lh7 = "-lh7-"
-            $lzh_magic_lh8 = "-lh8-"
-            $lzh_magic_lhd = "-lhd-"
-        condition:
-            any of them
-    """
-
-    YARA_MATCH_OFFSET = -2
+    PATTERNS = [
+        Regex(r"-lh0-"),
+        Regex(r"-lzs-"),
+        Regex(r"-lz4-"),
+        Regex(r"-lh1-"),
+        Regex(r"-lh2-"),
+        Regex(r"-lh3-"),
+        Regex(r"-lh4-"),
+        Regex(r"-lh5-"),
+        Regex(r"-lh6-"),
+        Regex(r"-lh7-"),
+        Regex(r"-lh8-"),
+        Regex(r"-lhd-"),
```

Comment:
HyperScan might internally produce a similar output:
```py
Regex(r"""
    -lzs- |
    -lz4- |
    -lh0- |
    -lh1- |
    -lh2- |
    -lh3- |
    -lh4- |
    -lh5- |
    -lh6- |
    -lh7- |
    -lh8- |
    -lhd-
    """, re.VERBOSE)
```

Reply:
indeed it will be the something similar, I would just keep it for easier readability